Documentation/CRDs/Cluster/external-cluster/.pages

```diff
@@ -1,3 +1,7 @@
 nav:
     - external-cluster.md
     - topology-for-external-mode.md
+    - advance-external.md
```

Comment:
Let's have these topics match the order that the user will need to run them. I would think the following order, and possibly rename. Seems like the topology topic should be last in the list.
- external-cluster.md
- provider-cluster.md --> provider-export.md
- consumer-cluster.md --> consumer-import.md
- upgrade-external.md
- topology-for-external-mode.md